content/en/blog/_posts/comparing-local-kubernetes-development-tools.md

```diff
@@ -0,0 +1,45 @@
+# Comparing Local Kubernetes Development Tools: Telepresence, Gefyra, and mirrord
```

Comment:
Please add valid blog article front matter (have a look at published blog articles for an example).